### PR TITLE
Export contract's storage to json

### PR DESCRIPTION
**Pull request by shargon**
# Description

Related to https://github.com/neo-project/neo/issues/3127

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: src/Neo.CLI/CLI/MainService.Blockchain.cs [hunk 1]
```diff
@@ -10,16 +10,58 @@
...
+            }
+
+            var prefix = new byte[4];
+            BinaryPrimitives.WriteInt32LittleEndian(prefix, contract.Id);
+
+            // Get all entries
+            var jsonData = new JObject();
+            foreach (var entry in snapshot.Find(prefix, Persistence.SeekDirection.Forward))
+            {
+                jsonData[Convert.ToBase64String(entry.Key.Key.ToArray())] = Convert.ToBase64String(entry.Value.Value.ToArray());
```
Comment (cschuchardt88) [inline src/Neo.CLI/CLI/MainService.Blockchain.cs:54]: I believe there is a limit in json that doesn't allow super long properties.

Comment (shargon) [reply src/Neo.CLI/CLI/MainService.Blockchain.cs:56]: We don't have extremely big data because of gas

Comment (shargon) [reply src/Neo.CLI/CLI/MainService.Blockchain.cs:56]: > in ASP.Net, there's "MaxJsonLength". The default is 2097152 characters, which is equivalent to 4 MB of Unicode string data.

I think that it's enough, and in our json version, we don't have this limit.

Comment (cschuchardt88) [reply src/Neo.CLI/CLI/MainService.Blockchain.cs:56]: I think its 3 characters per one byte of base64 encode. Isn't it?

Comment (Jim8y) [reply src/Neo.CLI/CLI/MainService.Blockchain.cs:56]: 4 bytes long for 3 bytes charadters.

**Comment by shargon** [comment]
It will me moved to a testing plugin

**Pull request closed without merging**